Src/Newtonsoft.Json/Converters/UnixDateTimeConverter.cs

```diff
@@ -36,6 +36,64 @@ public class UnixDateTimeConverter : DateTimeConverterBase
     {
         internal static readonly DateTime UnixEpoch = new DateTime(1970, 1, 1, 0, 0, 0, DateTimeKind.Utc);
 
+        private bool _allowPreEpoch;
+
+        /// <summary>
+        /// Gets or sets the default value of the <see cref="AllowPreEpoch"/>
+        /// property for new instances of the <see cref="UnixDateTimeConverter"/>
+        /// class.
+        /// </summary>
+        /// <value>
+        /// <c>true</c> to allow converting dates before Unix epoch to and from
+        /// JSON;
+        /// <c>false</c> to throw an exception when a date being converted to or
+        /// from JSON occurred before Unix epoch.
+        /// The default is <c>false</c>.
+        /// </value>
+        public static bool DefaultAllowPreEpoch { get; set; } = false;
```

Comment:
Static configuration like this isn't desirable. It is global and different apps/libraries can override each other. Please remove.